[ADD] sales: quotations

dikd-odoo opened the pull request:
New overview doc for sales quotations
-brief intro to sales quotations
-cards framework for quotations flow
-redirect all send_quotations to new sales_quotations doc

Target: 18.0+

Task: https://www.odoo.com/odoo/my-tasks/3809626

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/12202.png)](https://mergebot.odoo.com/odoo/documentation/pull/12202)

dikd-odoo commented:
Hi @Felicious this is ready for peer review, when you have a chance. Thank you!

Felicious commented on content/applications/sales/sales/sales_quotations.rst:
```diff
@@ -0,0 +1,112 @@
+:hide-page-toc:
+
+================
+Sales Quotations
```
```suggestion
Sales quotations
```

Felicious commented on content/applications/sales/sales/sales_quotations.rst:
```diff
@@ -0,0 +1,112 @@
+A *sales quotation* or quote is a document sent to a customer that outlines the estimated costs and
+terms for goods and services.
+
+In Odoo, sales quotations are configured in the **Sales** app.
```
I noticed "sales quotations" is used throughout the doc, but in Odoo, we generally refer to them as "quotations." To stay consistent with the UI naming, would it make sense to remove "sales" and just use "quotations" instead?

```suggestion
In Odoo, quotations are configured in the **Sales** app.
```

Felicious replied:
To continue this paragraph, let's also summarize how quotes are related to a number of workflows in Odoo, namely:
- How are quotes related to sales orders?
- We can configure them manually in the sales app, but where else can they be created? (e.g. CRM, Helpdesk, possibly others?) no need to go into it in this PR, but it might be worth to gloss over it to link to other docs.
-

Felicious commented on content/applications/sales/sales/sales_quotations.rst:
```diff
@@ -0,0 +1,112 @@
+.. image:: sales_quotations/sales_quotation.png
+   :alt: Unconfirmed sales quotation in Odoo **Sales** app.
+
+.. cards::
```
Instead of smushing all the cards at the bottom, let's move cards that are relevant to each section of your overview doc to put doc links to related sections in each place. 

For example, under "Sales quotations in business deals", we can put quote templates, optional products, online signatures, PDF quote builder? To me, these articles seem most relevant to the use case you're describing. I'll leave the judgement up to you!

dikd-odoo replied:
As discussed in earlier convo, Moved up the cards to the top, but kept them in a group to match the [accounting documentation](https://www.odoo.com/documentation/18.0/applications/finance/accounting.html) model. For now I think it might make more sense to keep them together as a sort of TOC/choose your own adventure, but will defer to your next review/final review.

StraubCreative replied:
I like that the cards are up at the top and closely mirror the Accounting success model.

We can dial this in more in a future PR to only include cards that are relevant to certain sections or to only include a "core" (highly common) multi-app flow (like CRM --> Sales --> Inventory --> Invoicing)

Felicious reviewed (changes requested):
Good work so far, @dikd-odoo
 
My main question for you is: What’s the goal of this overview doc?

Typically, an overview like this should also give a big-picture look at key workflows related to quotations—such as quote → sales order → invoice or opportunity → quote → sales order—so users can see how quotations fit into the broader sales process.

Your doc already does a great job of breaking down what a quotation includes. Adding a bit more context on these workflows would really tie everything together. Let’s talk through some ideas in our 1:1 today! I don’t think many changes will be needed, and after a quick second review, I’m pretty sure it’ll be good to go for approval. 😊

dikd-odoo commented:
Hi @Felicious thank you for the super thoughtful review! I implemented your changes and added some more context to the use case and where quotations fit in. This is ready for another look when you have a chance. Thank you!

Felicious reviewed (approved):
LGTM, @dikd-odoo !

dikd-odoo commented:
Hello @odoo/us-doc-review this is ready for final review, when you have a chance. Thank you!

StraubCreative commented:
@samueljlieber can you TR this please before i do final content review? 
The commit diff is substantial and I want to make sure we're kosher there.
TIA!

StraubCreative commented:
[c6d29c2](https://github.com/odoo/documentation/pull/12202/commits/c6d29c2844a3dda6b1c4e51d8ea10891ac0e2784): resolved merge conflict in `redirects.txt` (combined both the redirects here in the PR and from recent PR that moved UoM doc)

StraubCreative commented on content/applications/sales/sales/sales_quotations.rst:
```diff
@@ -0,0 +1,138 @@
+
+A *sales quotation* or quote is a document sent to a customer that outlines the estimated costs and
+terms for goods and services. Once accepted, a quotation can be converted into a sales order, which
+serves as the final agreement before delivery and invoicing.
```
This is a good intro, I like that you explicitly define what we're talking about in the doc (vs. "selling" Odoo, corp jargon, etc.)

My (optional) suggestion here is to use the following educational learning formula to add more/relevant context:

- Define the feature (or process)
- Frame the feature (or process) in the relevant business context/problem
- and then say how the feature behaves in Odoo. 

Example of the formula applied in the intro.

Do you think this addition makes the intro stronger?

```suggestion
serves as the final agreement before delivery and invoicing.

In Odoo, quotations are handled in the **Sales** application, and are composed of a form that
contains two main sections:

- a *domain* area where customer information is placed (such as name, address, contact info, etc.),
  sales templates and invoicing policies are assigned, and similar criteria is set.
- an *order lines* section where specific products are added to the quote, along with their
  quantities and any discounts (if applicable)

When a quotation is confirmed, it becomes a sales order. The sales order is a key stage in Odoo
since all revenue-generating and supply-chain operations hinge on this document. Many second-order
consequences connect to the sales order status and include a volume of useful automation options
(such as reordering rules in the Purchase application).

Read on to discover the most common configurations and use-case flows for sales quotations.

```
-

StraubCreative commented on content/applications/sales/sales/sales_quotations.rst:
```diff
@@ -0,0 +1,138 @@
+Quotations fit into a broader sales flow that connects different stages of customer interactions
+from initial interest to payment.
+
+The typical flow follows these steps:
```
This is a good start but is only partially true.

It depends on the business type (b2b, b2c) and if integrations like ecom, pos, etc. are integrated.

In eCom, for example, the quotation is instantly created and confirmed once payment is captured...the invoice comes later and delivery is last.

StraubCreative commented on content/applications/sales/sales/sales_quotations.rst:
```diff
@@ -0,0 +1,138 @@
+In Odoo, quotations are configured in the **Sales** app. They can also be generated from other apps
+as part of the sales workflow:
+
+- CRM: :doc:`Convert opportunities <../crm/acquire_leads/send_quotes>` into quotes to follow up on
```
This list is a good start. 

We're missing supply-chain and accounting apps, though, which are 
a) arguably the most important in Odoo, generally speaking 
b) and the most relevant in this context since business ops are closely integrated with the varying stages of quotes/SOs.

StraubCreative commented on content/applications/sales/sales/sales_quotations.rst:
```diff
@@ -0,0 +1,138 @@
+
+      Add custom PDF files to quotations to elevate the document's headers and designs.
+
+Sales quotations in business deals
```
Would cut this section
It looks/sounds important but doesn't contain any actionable knowledge. It's way better that we "show" vs. "tell,"  generally speaking, and also especially on user docs.

StraubCreative commented on content/applications/sales/sales/sales_quotations.rst:
```diff
@@ -0,0 +1,138 @@
+payment terms; and providing a documented stage where the business deal can be negotiated before
+agreed upon.
+
+Key components of a sales quotation
```
I love this section, this is an excellent example of how to "show" (vs. tell) and lead with a strong success model both in written and visual form. Very nice!

StraubCreative reviewed (approved):
Hi @dikd-odoo 
Approving with some comments to address on a future [IMP] PR.
I'm very glad we have a quotes overview doc now— thank you for making it!

@samueljlieber if you want to give another quick TR since I fixed the merge conflict on the redirects file, thanks!

samueljlieber reviewed (approved):
Hi @dikd-odoo! I made two very small fixes in [my last commit](https://github.com/odoo/documentation/compare/c6d29c2844a3dda6b1c4e51d8ea10891ac0e2784..9a17adda4c9e2a7ab33284d5d2911669777ecbe2#diff-8ba3c6c7a74557c344d7b2edb81a54ac717ad492ca25be44a8b3720f67966e51), to just remove an extra space and bold app names. I am approving now and will merge once checks pass. 

Thank you!

samueljlieber commented:
.....
@robodoo delegate=dikd-odoo

dikd-odoo commented:
Thank you both @StraubCreative @samueljlieber for the thorough and helpful final and technical reviews, as well as the tips for future PRs!
